.travis.yml

```diff
@@ -10,5 +10,5 @@ env:
     - ROBOT_ENVIRONMENT=test
 
 rvm:
-  - 2.3.7 # deployed
   - 2.4.4
```

Comment:
Can we get rid of 2.4 also?

Reply:
oops, missed before I merged, sorry

i think the answer would have yes, though